src/Tribe/Cost_Utils.php

```diff
@@ -449,4 +450,12 @@ private function filter_numeric_values( $value ) {
 	private function restore_original_decimal_separator( $value ) {
 		return str_replace( '.', $this->_current_original_cost_separator, $value );
 	}
+
+	private function numerize_numbers( $value ) {
```

Comment:
Ok, will do.
For future reference: are `private` and `protected` methods too to be documented?